docs/upgrading.md

```diff
@@ -274,7 +274,7 @@ React 17 introduced a new [JSX Transform](https://reactjs.org/blog/2020/09/22/in
 
 ## Upgrading from version 9 to 10
 
-There were no breaking changes between version 9 and 10.
+There were no breaking changes between versions 9 and 10.
```

Comment:
```suggestion
There were no breaking changes between version 9 and 10.
```

Paraphrasing, "version 9 and _version_ 10", but I think the current version is correct.